deviceaccess.py

```diff
@@ -815,24 +885,37 @@ class Device:
         np.bool_: "boolean"
     }
 
-    def __init__(self, aliasName: str = None) -> None:
+    def __init__(self, aliasName: Optional[str] = None) -> None:
         self.aliasName = aliasName
-        if aliasName:
-            self._device = pb.getDevice(aliasName)
-        else:
+        if aliasName is None:
             self._device = pb.getDevice_no_alias()
+        else:
+            self._device = pb.getDevice(aliasName)
 
     def __enter__(self):
-       """Helper function for with-statements"""
-       if self.aliasName is None:
-           raise SyntaxError('In a with-statement, an alias has to be provided in the device constructor!')
-       else:
-           self._device.open(self.aliasName)
-           return self
+        """Helper function for with-statements"""
+        if self.aliasName is None:
+            raise SyntaxError('In a with-statement, an alias has to be provided in the device constructor!')
+        else:
+            self._device.open(self.aliasName)
+            return self
 
     def __exit__(self, *args):
-       """Helper function for with-statements"""
-       self._device.close()
+        """Helper function for with-statements"""
+        self._device.close()
+
+    def get_catalogue_metadata(self, meta_tag: str) -> str:
```

Comment:
To be consistent with the rest of the class this should be ```getCatalogueMetadata()```. We use the camel case syntax here to be consistent with the C++, and deviate from the python style.

Reply:
While I would prefer to adopt Python Style Guide for all classes, methods and members, this would break the interface. Thus I changed the spelling of the method to camel-case, to maintain consistency within the existing interface, as requested. Change added with commit 030cc4d7b086e7e13b2468eae548a866471256f3.